### ISSUE TITLE
Load modules command?

### ISSUE THREAD

**Issue by justinwoo** [nwolverson/vscode-ide-purescript#119]
It would be nice to do load module / load all modules, e.g.

https://github.com/purescript-emacs/psc-ide-emacs/blob/a10cc85565f330ee277698b27f3f715fef2e1ce2/psc-ide-protocol.el#L80-L87

Not entirely sure where to start looking on this.

**Comment by nwolverson** [comment]
Yeah right now this is triggered on start (or restart ide server command) and on completion of a full build. I always thought about adding this but as restart command works quickly enough I never bothered.

Probably a minor copy-paste and deleting code in the language server to expose this, here's where it's done on restart:

https://github.com/nwolverson/purescript-language-server/blob/master/src/LanguageServer/IdePurescript/Main.purs#L145
